### ISSUE TITLE
Model summary? or other way to see shape of all layers in a model.

### ISSUE THREAD

**Issue by wjessup** [ml-explore/mlx#175]
Option to do:

for params in MyNet.parameters():
    print(f'{params} {MyNet[params]["weight"].shape }')

should add to documentation or make an easier method.

**Comment by awni** [comment]
If you just print the model it should show you the layers with their arguments. If the pretty print is missing output for some layers let us know:

`print(MyNet)`

Alternatively if you want individual arrays with shapes you can do:

```
from mlx.utils import tree_map
shapes = tree_map(lambda p: p.shape, model.parameters())
```

**Comment by wjessup** [comment]
Thumbsup. Add to documentation?

**Comment by awni** [comment]
Sure, which command did you find useful, the first or the second (or both)?

**Comment by wjessup** [comment]
both, with an example to show what the output looks like.

**Comment by awni** [comment]
Ok will update docs with a bit more detail on that.

**Comment by awni** [comment]
Addressed in #198
